.gitlab-ci.yml

```diff
@@ -42,6 +42,10 @@ RPM:
           - aws/centos-stream-9-aarch64
           - aws/rhel-8.5-ga-x86_64
```

Comment:
These need to be removed, otherwise we will run into conflicts. They use the same URL as rhel-8.6-ga so it's fine to remove them.